docs/learning/WriteAConnector.md

```diff
@@ -62,16 +61,12 @@
     - [Locks & Codes](#locks--codes)
     - [More information](#more-information)
 
-## New ConnectorFramework
-
-A new ConnectorFramework dependent upon iTwin.js 3.x is forthcoming early in 2022. The old iModelBridge code delivered with old iModel.js mono repo has been deprecated and removed. It was moved to its own repository and will be available in its own npm package.
-
 ### Porting a connector
 
 To port an existing Connector originally written on iModel.js 2.x, it will be necessary to:
 
-1. Change the dependencies to the [New Connector Framework](#new-connectorframework)
-2. Search and replace "iModelBridge" with "BaseConnector", "BridgeJobDefArgs" with "ConnectorJobDefArgs" and "BridgeRunner" with "ConnectorRunner" and so on.
+1. Change the dependencies to the latest version.
```

Comment:
latest version of what

Reply:
> having a preview site of doc changes in the ci would be kinda neat @williamkbentley @ben-polinsky

I assume it can be done one way or another.

Reply:
There is a CLI package available that allows you to build the doc site locally, if that would help. Other than that, the CI job does build the docs to validate links. It doesnt do anything with the output. I suppose we could put it somewhere.

Reply:
Yeah we could just publish the output and download it since it is static or push it to netlify or something.

Reply:
@aruniverse just elaborated on which version of what.  Hope that makes better sense now.